modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PhpZendExpressivePathHandlerServerCodegen.java

```diff
@@ -291,46 +292,40 @@ protected String generateUniqueDefinitionName(String name, OpenAPI openAPI) {
         objs = super.postProcessOperationsWithModels(objs, allModels);
         Map<String, Object> operations = (Map<String, Object>) objs.get("operations");
         List<CodegenOperation> operationList = (List<CodegenOperation>) operations.get("operation");
-        String interfaceToImplement;
-        StringBuilder interfacesToImplement = new StringBuilder();
-        String classMethod;
+        String httpMethodDeclaration;
         String pathPattern = null;
         for (CodegenOperation op : operationList) {
             switch (op.httpMethod) {
                 case "GET":
-                    interfaceToImplement = "Operation\\GetInterface";
-                    classMethod = "handleGet";
+                    httpMethodDeclaration = "Get()";
                     break;
                 case "POST":
-                    interfaceToImplement = "Operation\\PostInterface";
-                    classMethod = "handlePost";
+                    httpMethodDeclaration = "Post()";
                     break;
                 case "PATCH":
-                    interfaceToImplement = "Operation\\PatchInterface";
-                    classMethod = "handlePatch";
+                    httpMethodDeclaration = "Patch()";
                     break;
                 case "PUT":
-                    interfaceToImplement = "Operation\\PutInterface";
-                    classMethod = "handlePut";
+                    httpMethodDeclaration = "Put()";
                     break;
                 case "DELETE":
-                    interfaceToImplement = "Operation\\DeleteInterface";
-                    classMethod = "handleDelete";
+                    httpMethodDeclaration = "Delete()";
                     break;
                 default:
-                    throw new RuntimeException("Unknown HTTP Method " + op.httpMethod + " not allowed");
+                    httpMethodDeclaration = "HttpMethod(\"" + op.httpMethod + "\")";
             }
-            if (interfacesToImplement.length() > 0) {
-                interfacesToImplement.append(", ");
+            op.httpMethod = httpMethodDeclaration;
+            //Producing content with media type "*/*" is not supported
+            if (op.produces != null) {
+                for (Map<String, String> p: op.produces) {
+                    p.replace("mediaType", "*/*", "n/a");
```

Comment:
I believe it is a bit excessive and unnecessary - `php-ze-ph` already generates "TODO" for code fragment that uses "produces":
```
{{#produces}}
     * TODO check if producer is valid, if it has correct priority and if it can be moved to class annotation
     * @PHA\Producer(name=PHProducer\Transfer::class, mediaType="{{{mediaType}}}")
{{/produces}}
```
And personally I can not imagine a real API operation specification with `*/*` in response content declaration - that would mean empty response body because it seems slightly impossible to produce any media type that client accepts. Transformation to `n/a` was added just to prevent broken PHP-code generation for `modules\openapi-generator\src\test\resources\3_0\petstore-with-fake-endpoints-models-for-testing.yaml`. It would be so much simpler if "produce" was still a media type like [in OAS 2](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#operation-object), not media type or media type range like [in OAS 3](https://swagger.io/specification/#responseObject).

Reply:
> And personally I can not imagine a real API operation specification with */* in response content declaration 

I've seen spec using */* before. I'm not saying I encourage the use of */* but rather to inform the users so that they know what to fix. I'm sure new users of OpenAPI Generator will welcome the warning message to inform them what's been done behind the scene.